app/helpers/search_helper.rb

```diff
@@ -8,4 +8,69 @@ def view_more_text
       'website'
     end
   end
+
+  def access_level
+    if guest?
+      'guest'
+    else
+      'full'
+    end
+  end
+
+  # Current request IP is outside our campus range
+  def guest?
```

Comment:
Does this function really provide any utility? `access_level` could just use `member?`.